### DISCUSSION TITLE
user-guide/plugins/zotero-gpt

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#171] [文档评论区]
# user-guide/plugins/zotero-gpt

Awesome GPT for Zotero Awesome GPT for Zotero 是一个由 MuiseDestiny 创建的开源 Zotero 插件。这个插件使得用户能够通过命令标签使用 Meet API，进一步增强 Zotero 与 GPT 模型（包括 gpt-3.5-turbo 和 gpt-4）的集成。 项目主页：https://gith...

https://zotero-chinese.com/user-guide/plugins/zotero-gpt.html

**Comment by Freya1027** [comment]
为什么充了值还是会一直显示model-not-found 呀 TT

**Reply by Chikit-L** [reply]
常见问题里面有讲到

**Reply by Karllihaiq** [reply]
你好，请问解决了么，能够详细说下如何配置gpt4么，最好带着截图，API KEY MODEL是啥样子的，相关信息可以码掉。主要是不知道在哪找和填什么

**Reply by Chikit-L** [reply]
> 你好，请问解决了么，能够详细说下如何配置gpt4么，最好带着截图，API KEY MODEL是啥样子的，相关信息可以码掉。主要是不知道在哪找和填什么

你的问题在教程中的配置Zotero GPT章节全部都有提到。关于model的具体代码，需要向api的提供商查询。

**Reply by Karllihaiq** [reply]
谢谢您的解答，不过还是出现he model gpt-4 does not exist or you do not have access to it.

api https://api.openai.com
secretKey sk-…gucA
model gpt-4
temperature 1.0

secretKey是刚刚在openai里面生成的，是月付费用户，没有单独购买 key。请问还有哪些需要核对的

**Reply by Chikit-L** [reply]
> 谢谢您的解答，不过还是出现he model gpt-4 does not exist or you do not have access to it.
> 
> api https://api.openai.com secretKey sk-…gucA model gpt-4 temperature 1.0
> 
> secretKey是刚刚在openai里面生成的，是月付费用户，没有单独购买 key。请问还有哪些需要核对的

可以在https://openai.com/api/pricing/ 查询gpt model的模型代码，model gpt-4 其实是不对的。另外api应该和网页是分开付费的，这里我们并没有进行测试，您可以再确实一下是否网页付费用户是否可以调用Gpt4的模型。

**Reply by Karllihaiq** [reply]
谢谢您的意见。似乎是需要单独购买API。祝好~

**Comment by fuyueagain** [comment]
为什么安装了以后没有显示这个工具呢，zotero6版本

**Reply by Chikit-L** [reply]
快捷键能使用就没有问题

**Comment by tayloveshihan** [comment]
我只能通過快捷鍵進行打開，不能關閉，請問如何關閉該窗口？非常感謝

**Reply by Chikit-L** [reply]
通過“ESC”鍵關閉窗口

**Comment by Dsl5817055** [comment]
我的一直提示下面的信息，这是怎么回事啊？
出错了，下面是一些报错信息：
HTTP POST https://api.chatanywhere.com.cn/v1/v1/chat/completions failed with status code 404:{“timestamp”:1722061757426,“status”:404,“error”:“Not Found”,“path”:“//v1/chat/completions”}
如果是网络问题，建议优化网络，比如打开/切换代理等。

**Reply by SYangHE** [reply]
我也是这样

**Reply by Chikit-L** [reply]
> ####

无法连接到 api.chatanywhere。请先排除本地的网络问题。

**Comment by yijiaochuaifei** [comment]
请问为什么用官方的secretkey，才申请，我看usage也显示没花钱5刀的限额，还是显示
insufficient_quota
You exceeded your current quota, please check your plan and billing details.

**Comment by qiangfengchuifu-xy** [comment]
想请问zotero7.0版本，安装GPT插件后，设置界面找不到GPT，也就无法配置，是什么情况呢？

**Reply by Chikit-L** [reply]
应该是插件加载失败。尝试重启电脑。

**Comment by hhao659** [comment]
你好我想请问一下关于AskGpt无法直接访问文章内容的问题如何解决？

**Reply by Chikit-L** [reply]
必须在zoterogpt的输入框中输入一些内容（也就是必须有提问的问题），再点击下方的标签（例如AskPDF）。若输入框为空（相当于你没问任何问题，它自然不知道要干什么），则不会解析命令语句。

例如你可以输入本文创新点、解决的问题等等

**Comment by qiangfengchuifu-xy** [comment]
我也还没解决啊，哭了

--------------原始邮件--------------
发件人："hhao659 ***@***.***&gt;;
发送时间：2024年9月5日(星期四) 中午11:02
收件人："zotero-chinese/wiki" ***@***.***&gt;;
抄送："肖逸 ***@***.***&gt;;"Comment ***@***.***&gt;;
主题：Re: [zotero-chinese/wiki] user-guide/plugins/zotero-gpt (Discussion #171)
-----------------------------------

 

 
你好我想请问一下关于AskGpt无法直接访问文章内容的问题如何解决？

**Comment by Phphoenixx** [comment]
我下了最新的zotero7.03和最新的zotero-gpt，安装完插件后还是找不到打开的图标，快捷键control/也没办法打开，重启之后还是这样

**Reply by Chikit-L** [reply]
尝试禁用其他插件或者重启，如果都没有效果，请到插件的GITHUB主页向作者反馈。

**Comment by WPengXiang** [comment]
如何使用kimi接口

**Reply by Chikit-L** [reply]
插件的GITHUB主页有相关讨论。

**Comment by Charliez-929** [comment]
有没有出现zotero7打开翻译插件gpt插件就不能使用的情况，ctrl/不能打开，也无法配置

**Reply by xxzz-de** [reply]
我也是这样的

**Reply by Charliez-929** [reply]
把那几个插件开开关关就突然又能用了

**Comment by Coddoob** [comment]
GPT命令框不能够鼠标来进行拖动吗？

**Comment by githubliu818** [comment]
The URI is malformed 怎么解决啊

**Comment by SEKI-2025** [comment]
为什么输入了问题后，点击AskPDF会出现弹窗[JavaScript Application] missing )in parenthetical。然后也没办法正常回答我的问题，检查了一下代码好像没问题

**Reply by Chikit-L** [reply]
https://github.com/MuiseDestiny/zotero-gpt/issues/347#issuecomment-2310204181

**Comment by gjxzyf** [comment]
为什么快捷键CTRL + R 和CTRL + S没有反应呢

**Comment by papavera** [comment]
Meet.Global.views.messages[0] is undefined这是为啥啊

**Reply by Chikit-L** [reply]
必须在zoterogpt的输入框中输入一些内容（也就是必须有提问的问题），再点击下方的标签（例如AskPDF）。若输入框为空（相当于你没问任何问题，它自然不知道要干什么），则不会解析命令语句。

例如你可以输入本文创新点、解决的问题等等

**Comment by contsinrr** [comment]
想问一下zotero gpt现在支持国产大模型吗？

**Reply by Chikit-L** [reply]
支持，具体操作请在https://github.com/MuiseDestiny/zotero-gpt/issues 进行搜索。

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by JertaHe** [comment]
视窗可以移动吗？挡住正文了但是不知道怎么移动有点恼火啊😂

**Reply by Chikit-L** [reply]
按住`ctrl`

**Comment by erdan9** [comment]
askPDF的时候，文献太长，GPT阅读不了怎么办

**Comment by six-0317** [comment]
出错了，下面是一些报错信息：
HTTP POST https://api.openai.com failed with status code 421:
{
"message": "Welcome to the OpenAI API! Documentation is available at https://platform.openai.com/docs/api-reference"

}
如果是网络问题，建议优化网络，比如打开/切换代理等。
full api https://api.openai.com
secretKey sk-…pxsA
model gpt-3.5-turbo
temperature 0
这是怎么回事啊

**Reply by Chikit-L** [reply]
网络问题，建议优化网络，比如打开/切换代理等。

**Reply by six-0317** [reply]
我切换了代理也不太行哇

**Reply by six-0317** [reply]
会是我设置的问题么？

**Reply by Chikit-L** [reply]
输入`/report`，正常情况会显示类似于
```
api https://api.chatanywhere.tech
secretKey sk-…4YCN
model gpt-3.5-turbo
temperature 1.0
```
你这里显示了`full api https://api.openai.com/ `，这里的**full**看起来是多余的，但我还是认为这个报错更有可能是网络问题引起的。

**Reply by six-0317** [reply]
HTTP POST https://api.openai.com/v1/chat/completions failed with status code 429:

{

"error": {

    "message": "You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.",

    "type": "insufficient_quota",

    "param": null,

    "code": "insufficient_quota"

}

}

如果是网络问题，建议优化网络，比如打开/切换代理等。

full api https://api.openai.com/v1/chat/completions

secretKey sk-…pxsA

model gpt-3.5-turbo

temperature 1


现在又变成了这个，麻烦您指点

**Reply by Chikit-L** [reply]
https://zotero-chinese.com/user-guide/plugins/zotero-gpt#you-exceeded-your-current-quota

**Comment by control-spiderman** [comment]
想请问下自建代理的，我的代理需要怎么配置才能匹配上这个项目？

**Reply by Chikit-L** [reply]
需要api以及secretKey。具体的请咨询所使用的GPT提供方

**Comment by psyduck1112** [comment]
请问怎么渲染回答中的公式

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by hl714** [comment]
为什么有的文章能用，有的文章用不了？

**Comment by wenwenbang** [comment]
用了半天没明白要怎么把gpt的回答复制下来，当回答里面有很多公式的时候，公式就选中不了了。有办法直接复制回答的全文吗？

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by onewciyo** [comment]
请问：问题回答到一半暂停了，也无法显示完整回答，如何操作能够刷新看到以前提问的完整回答呢？

**Comment by cincofish** [comment]
为什么zotero的save as note一直用不了呀

**Comment by eecoder-dyf** [comment]
怎么移动窗口位置？用鼠标拖动没用

**Reply by Chikit-L** [reply]
> 怎么移动窗口位置？用鼠标拖动没用

换几个位置拖拽

**Reply by ramonmi** [reply]
摁住Ctrl并拖动

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by Halabouqii** [comment]
请问运行UploadPDF时出现以下报错，如何解决？

UploadPDF execution error: window[_0xd36cf(...)][_0xd36cf(...)] is undefined

(async () => {
  const pdfItems = []
  for (let item of ZoteroPane.getSelectedItems()) {
    const pdfItem = item.isPDFAttachment() ? item : await item.getBestAttachment()
    pdfItem && pdfItems.push(pdfItem)
  }
  await window.Meet.OpenAI.uploadFile(pdfItems)
  return ""
})()

---
Please check the code inside the command tag.

**Reply by windbd** [reply]
我也是

**Comment by mia920909** [comment]
您好，我想问下，复制markdown的笔记后超链接会消失，只能自己再插入。有什么办法解决吗？谢谢！

**Comment by ArdenZhao0825** [comment]
HTTP POST https://api.openai.com/v1/chat/completions failed with status code 429:
{
“error”: {
“message”: “Your access was terminated due to violation of our policies, please check your email for more information. If you believe this is in error and would like to appeal, please contact us through our help center at help.openai.com.”,
“type”: “access_terminated”,
“param”: null,
“code”: “access_terminated”
}
}
请问为什么会显示这样的报错啊TT

**Comment by EliauKaiJ** [comment]
今天一直显示报错：🎊 出错啦，不要灰心～
下面是报错信息：

Request timed out after 30000 ms
切换了网络还是不行

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by DeepNight-MING** [comment]
有办法彻底删除Awesome GPT的插件和其配置文件（即重置），再重新安装吗？
尝试了在插件管理器里面卸载后再重装，还是原来的配置，由于我是从zotero6升级到7的，里面的按钮基本都没法用，比如item、askPDF都失效，怀疑是配置文件有问题

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by Weihao6667** [comment]
为什么我按照教程，充值成功了GPT-4的API，在zotero中依旧显示The model GPT-4o does not exist or you do not have access to it.

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by zz9915** [comment]
出错啦，不要灰心～
下面是报错信息：

HTTP POST https://api.chatanywhere.tech/v1/chat/completions failed with status code 401:
{“error”:{“message”:“ApiKey错误(当前请求使用的ApiKey: sk-pro****dKgA)。wrong api key”,“type”:“chatanywhere_error”,“param”:null,“code”:“401 UNAUTHORIZED”}}

⭐️ 请截图寻求帮助，注意保持礼貌～ q请问这是啥原因啊。。

**Comment by su-chun** [comment]
为什么有的pdf打开上边栏又gpt按钮，但是有的打开没有，有的刚开始打开有，又会突然没有？

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by yuyu3919** [comment]
出现model not allow 的原因是什么呀
HTTP POST http://llmapi.paratera.com/v1/chat/completions failed with status code 405:
{“detail”:“Method Not Allowed”}
url http://llmapi.paratera.com/v1/chat/completions
secretKey sk-…RuGw
model DeepSeek-R1
temperature 1
maxTokens 4096

**Reply by Chikit-L** [reply]
咨询api提供商

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

[a comment by an automated account is not shown]

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by momo-ying** [comment]
各位理工科大佬们，请问为什么会出现这种情况呀？sos，救救小白吧，谢谢谢谢，非常感谢，祝科研顺利！！！

出错啦，不要灰心~
下面是报错信息:
HTTP POST 请求失败，状态码为 404:
{"code":5,"error":"url.not found","message":"没找到对应项"}
象","方法":"POST","错误码":"0x5","状态":false,"用户代理":"Mozila/5.0(Windows NT 10.0; Win64.
x64; rv:109.0) Gecko/20100101 Firefox/115.0""网址":"/v1/v1/chat/completions"!
请截图寻求帮助，注意保持礼貌~

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

[a comment by an automated account is not shown]

**Comment by Vc-86** [comment]
请问行内公式无法渲染是为什么啊？比如 $ m $这种，单行的公式可以正常显示

**Comment by woyaokaoyanhaha** [comment]
🎊 出错啦，不要灰心～
下面是报错信息：
> HTTP POST https://linuxdoapi-api-wow.223387.xyz/v1/chat/completions failed with status code 400:
>
>{"error":{"message":"Conversation roles must alternate user/assistant/user/assistant/... Conversation roles must alternate user/assistant/user/assistant/...","type":"BadRequestError","param":"","code":400}}

⭐️ 请截图寻求帮助，注意保持礼貌～


`url` https://linuxdoapi-api-wow.223387.xyz/v1/chat/completions
`secretKey` sk-...6uh5
`model` gemma-3-27b-it
`temperature` 1
`maxTokens` 4096             请问这是什么情况

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by chanyanaidai** [comment]
这个窗口不能移动吗？每次问问题都要重新读取全文好烧token啊

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by marshallfan** [comment]
啥时候能接入deepseek v4 Pro啊，现在只有v4 flash感觉比v3.2提升不大

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！

**Comment by yezhiyi9670** [comment]
这个插件并非“开源”，Github 上的代码只是个幌子，[已经很久没有更新任何实质内容](https://github.com/MuiseDestiny/zotero-gpt/commits/3.1.4/)，package.json 里的版本号还是 0.2.9，并且插件商店所使用的 Gitee 下载源仓库[明确说明“是商业插件，不可未经许可分发或修改”](https://gitee.com/MuiseDestiny/plugins)。

**Comment by contsinrr** [comment]
邮件已收到，谢谢您的配合！
